packages/tx/test/legacy.spec.ts

```diff
@@ -445,7 +452,7 @@ tape('[Transaction]', function (t) {
   t.test('sign(), verifySignature(): sign tx with chainId specified in params', function (st) {
     const common = new Common({ chain: 42, hardfork: Hardfork.Petersburg })
     let tx = Transaction.fromTxData({}, { common })
-    st.ok(tx.common.chainId().eqn(42))
+    st.ok(tx.common.chainId() === BigInt(42))
```

Comment:
some more st.ok cases here we can use st.equals, looks like there are a lot in the tx spec files, i can do a more comprehensive search and replace sweep later if you'd like

Reply:
Yep, I'm sure there are a lot of places where I missed that. I'm planning to go back and do more of that today. I started making those changes along the way based on your feedback yesterday and am planning to work on all the cosmetic stuff once I get the VM tests all passing.  I'll put together a list of what all needs to change still later this morning and start working through it. Thanks for the review!